### PR TITLE
Use `ci-test` alias

### PR DESCRIPTION
**Pull request by DaughterOfMars**
# Description of change

Use the github alias in the test CI.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

### CODE REVIEW

File: .cargo/config.toml [hunk 1]
```diff
@@ -7,6 +7,8 @@ ci-check-nostd = "check --no-default-features -F serde -p iota-sdk --target risc
 ci-check-types = "check --no-default-features -p iota-sdk"
 
 ci-test = "nextest run --all-features --profile ci --cargo-profile ci -p iota-sdk -p iota-sdk-bindings-core"
+ci-tangle-test = "nextest run --tests --all-features --run-ignored ignored-only --profile ci --cargo-profile ci -p iota-sdk -p iota-sdk-bindings-core"
+ci-coverage = "llvm-cov nextest --lcov --output-path coverage.info --tests -p iota-sdk --all-features --run-ignored all --profile ci"
```
Comment (thibault-martinez) [inline .cargo/config.toml]: wen fix coverage

File: .github/workflows/coverage.yml [hunk 2]
```diff
@@ -73,5 +65,4 @@ jobs:
         uses: coverallsapp/github-action@v2.2.1
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          path-to-lcov: coverage.info
```
Comment (thibault-martinez) [inline .github/workflows/coverage.yml:76]: Maybe temporarily enable coverage for this PR because I've tried that and it doesn't work

Comment (thibault-martinez) [reply .github/workflows/coverage.yml:76]: You can also try in my PR https://github.com/iotaledger/iota-sdk/pull/1035

File: .cargo/config.toml (hunk 1 above)
Comment (DaughterOfMars) [reply .cargo/config.toml]: now hehehe

**Review by thibault-martinez — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**